controllers/admin/AdminImportController.php

```diff
@@ -3629,6 +3625,13 @@ public function attributeImport($offset = false, $limit = false, &$crossStepsVar
         $regenerate = Tools::getValue('regenerate');
         $shopIsFeatureActive = Shop::isFeatureActive();
 
+        $processedProducts = [];
+        $truncateOnlyProductCombinations = false;
+
+        if (!$offset  && !$validateOnly && (($shopIsFeatureActive && $this->context->employee->isSuperAdmin()) || !$shopIsFeatureActive) && Tools::getValue('truncate')) {
+            $truncateOnlyProductCombinations = true;
```

Comment:
This commit changes semantics of `Delete all combinations before import` functionality.

We should keep that functionality in place, because some merchants actually might want to truncate all combinations before import new one. 

There should be separate option for this functionality in import settings

![image](https://github.com/user-attachments/assets/2d4b4269-8962-440f-bc2b-31ae6df560fb)

Reply:
new field added probably we will need translation etc